installer/charts/values.yaml.tpl

```diff
@@ -94,6 +98,16 @@ subscriptions:
 minIOOperator:
   enabled: {{ $minIOOperatorEnabled }}
 
+
+#
```

Comment:
We use this comment to document the chart that uses the value. I would argue that maybe you should create an `rhtap-odf` chart instead of adding the config to `rhtap-infrastructure`.

Reply:
I like the fact that I can control the resources we need for ODF without having to disable or enable a fool chart, and there is only two resources we need to generate. So not sure I agree, but could go either way if we think we will need to follow a different pattern we follow with minio today. The minio operator is getting installed via a helm chart vs odf via subscription

Reply:
I'm fine with the subscription being enabled in `values.yaml.tpl`.

Reply:
The comment about moving this to rhtap-infrastructure (since it's the chart that uses those values) still stands though.

Reply:
I still don't see the benefit of an additional chart. We are not doing that with minio storage, why do we want to do it with ODF storage. It is just infra dependency.

Reply:
Understand now the ask here, thanks for time on the call to clarify. Moving the odf vars to infrastructure object

Reply:
Done